components/accept-financial-contributions/ApplyToHost.js

```diff
@@ -0,0 +1,194 @@
+import styled from 'styled-components';
+import { graphql } from '@apollo/react-hoc';
+import { defineMessages, injectIntl } from 'react-intl';
+
+import { gqlV2 } from '../../lib/graphql/helpers';
+
+import { H1, H2, H3, P } from '../Text';
+import Container from '../Container';
+import StyledButton from '../StyledButton';
+import ExternalLink from '../ExternalLink';
+import HostCollectiveCard from './HostCollectiveCard';
+import Loading from '../Loading';
+
+const COLLECTIVE_CARDS_PER_PAGE = 5;
+const FISCAL_HOST_LINK = 'https://docs.opencollective.com/help/fiscal-hosts/become-a-fiscal-host';
+
+const AllCardsContainer = styled(Flex).attrs({
+  flexWrap: 'wrap',
+  width: '60%',
+  maxWidth: 1300,
+  mx: 'auto',
+  justifyContent: 'space-evenly',
+})``;
+
+const CollectiveCardContainer = styled.div`
+  width: 280px;
+  padding: 20px 15px;
+`;
+
+const InterestedContainer = styled(Container)`
+  box-shadow: 0 1px 3px 2px rgba(46, 77, 97, 0.1);
+  border-radius: 16px;
+  width: 20%;
+`;
+
+class ApplyToHost extends React.Component {
+  static propTypes = {
+    data: PropTypes.object,
+    collective: PropTypes.object,
+    intl: PropTypes.object.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.messages = defineMessages({
+      becomeHost: {
+        id: 'home.fiscalHost.becomeHostBtn',
+        defaultMessage: 'Become a fiscal host',
+      },
+      interestedInHosting: {
+        id: 'fiscalHost.interestedInHosting',
+        defaultMessage: 'Are you interested in fiscally hosting Collectives?',
+      },
+      header: {
+        id: 'acceptContributions.picker.header',
+        defaultMessage: 'Accept financial contributions',
+      },
+      applyToHost: {
+        id: 'pricing.applyFiscalHost',
+        defaultMessage: 'Apply to a fiscal host',
+      },
+      infoParagraph: {
+        id: 'fiscalHost.apply.info',
+        defaultMessage:
+          "With this option, you don't need to set up a legal entity and bank account for your project. The fiscal host will hold funds on your behalf, and take care of accounting, invoices, tax, admin, payments, and liability. Most hosts charge a fee for this service (you can review these details on the host's page before confirming).",
+      },
+    });
+  }
+
+  render() {
+    const { data, intl } = this.props;
+
+    if (!data.allHosts || !data.allHosts.collectives) {
+      return <div />;
+    }
+
+    const collectives = [...data.allHosts.collectives];
+
+    if (collectives.length === 0) {
+      return <div />;
+    }
+
+    return (
+      <Fragment>
+        <Box mb={4} mt={5}>
+          <H1 fontSize={['H5', 'H3']} lineHeight={['H5', 'H3']} fontWeight="bold" color="black.900" textAlign="center">
+            {intl.formatMessage(this.messages.header)}
+          </H1>
+        </Box>
+        <Container alignItems="flex-end" display="flex" pl={7} width={1}>
+          <img src="/static/images/create-collective/umbrellaPot.png" width="264px" height="184px" />
```

Comment:
I like using `import` for the images (like in [here](https://github.com/opencollective/opencollective-frontend/blob/bb5edbc723ba7cea23527986ad7028ecc6d8afac/components/expenses/ExpenseTypeRadioSelect.js#L11)).

Some reasons why I think it's better:
- The build system knows about the image, so if you delete it the build will fail (=> no 404 on images)
- A hash is added to the filename, so if we update it the cache will be updated properly
- We can globally configure strategies & optimization for our assets in the build
- If we do that for all our assets, it will be easier to find & remove the unused ones

That being said, it's not a general practice yet in our codebase so there's no requested changes but I'd like to raise the discussion. @opencollective/core-developers

Reply:
It's a good practice.

Reply:
Noted and changed!